### PR TITLE
feat(content-search-toolkit): New site search features

### PR DESCRIPTION
**Pull request by rafnarnason**
a few major changes to site search

- elasticsearch query highly tuned to make popular content score higher via relevance score
- fuzzy search to allow typos
- search results are highlighted on the search term context
- organization "frontpage" now included in results (demo /leit?q=*&type=webOrganizationPage)

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/SearchInput/SearchInput.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ import {
 import * as styles from './SearchInput.css'
 import { LinkType, useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
 import { TestSupport } from '@island.is/island-ui/utils'
+import { usePlausible } from '@island.is/web/hooks'
```
Comment (kksteini) [inline apps/web/components/SearchInput/SearchInput.tsx:43]: Is this import used?

Comment (rafnarnason) [reply apps/web/components/SearchInput/SearchInput.tsx]: nope - removing...

File: libs/api/domains/content-search/src/lib/contentSearch.service.ts [hunk 2]
```diff
@@ -74,12 +74,27 @@ export class ContentSearchService {
       query,
     )
 
+    // intercept highlights
+    let items = body.hits.hits.map((item) =>
```
Comment (kksteini) [inline libs/api/domains/content-search/src/lib/contentSearch.service.ts]: `const` ?

**Review by kksteini — APPROVED** [review]

File: libs/api/domains/content-search/src/lib/contentSearch.service.ts (hunk 2 above)
Comment (rafnarnason) [reply libs/api/domains/content-search/src/lib/contentSearch.service.ts]: fixed

**Review by RunarVestmann — APPROVED** [review]

**Pull request merged**